testcases/PowerNVDump.py

```diff
@@ -169,13 +170,14 @@ def setup_pwdless_auth(self):
             self.c.run_command(
                 "ssh-keygen -q -t rsa -f %s -P ''" % self.rsa_path)
             self.c.run_command("chmod 400 %s" % self.rsa_path)
-            self.c.run_command("sshpass -p %s ssh-copy-id -o \"StrictHostKeyChecking no\" -i %s root@%s" %
```

Comment:
The next line also should be removed else the test case will fail with syntax error.